### PR TITLE
added release pipeline

### PR DESCRIPTION
**Pull request by nitishsharma25**

### CODE REVIEW

File: .pipelines/release-pipeline.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+trigger:
```
Comment (rishky-msft) [inline .pipelines/release-pipeline.yml:1]: trigger: None

File: .pipelines/release-pipeline.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  inputs:
+    script: |
+      sudo npm install -g yarn
+      sudo npm install -g vsce
+- task: CmdLine@2
+  displayName: 'Building Vsix Packages'
+  inputs:
+    script: |
+      yarn install
+      yarn run compile
```
Comment (rishky-msft) [inline .pipelines/release-pipeline.yml:33]: Not required

Comment (nitishsharma25) [reply .pipelines/release-pipeline.yml]: removed

File: .pipelines/release-pipeline.yml (hunk 1 above)
Comment (nitishsharma25) [reply .pipelines/release-pipeline.yml]: done

**Review by rishky-msft — APPROVED** [review]

**Pull request merged**